ts/components/TosWebviewComponent.tsx

```diff
@@ -123,22 +119,18 @@ const TosWebviewComponent: React.FunctionComponent<Props> = ({
           )}
         />
       </View>
-      {shouldRenderFooter && !hasError && (
+      {shouldRenderFooter && onAcceptTos && (
         <FooterWithButtons
-          type={"TwoButtonsInlineThird"}
-          leftButton={{
-            block: true,
-            bordered: true,
-            onPress: onExit,
-            title: I18n.t("global.buttons.exit"),
-            testID: "toSWebViewContainerFooterLeftButton"
-          }}
-          rightButton={{
-            block: true,
-            primary: true,
-            onPress: onAcceptTos,
-            title: I18n.t("onboarding.tos.accept"),
-            testID: "toSWebViewContainerFooterRightButton"
+          type="SingleButton"
+          sticky={true}
+          primary={{
+            type: "Solid",
+            buttonProps: {
+              onPress: onAcceptTos,
+              label: I18n.t("onboarding.tos.accept"),
+              accessibilityLabel: I18n.t("onboarding.tos.accept"),
+              testID: "AcceptToSButton"
+            }
           }}
         />
```

Comment:
I see no gradient in the screenshot of the new screen on the JIRA ticket and on [Figma](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?type=design&node-id=5093-31126&mode=design&t=chsUiEhWva6NvQvf-4). Am I right?

As above, what about doing this?

```ts
        <ContentWrapper>
          <VSpacer size={8} />
          <ButtonSolid
            fullWidth
            onPress={onAcceptTos}
            label={I18n.t("onboarding.tos.accept")}
            accessibilityLabel={I18n.t("onboarding.tos.accept")}
            testID="AcceptToSButton"
          />
       </ContentWrapper>
```